src/fidesops/main.py

```diff
@@ -235,6 +235,10 @@ def read_ui_files(request: Request) -> FileResponse:
 
 def start_webserver() -> None:
     """Run any pending DB migrations and start the webserver."""
+    logger.warning(
+        "Fidesops has been deprecated. The codebase has merged into the Fides repo. Located at "
+        "https://github.com/ethyca/fides"
```

Comment:
The current warning message wording is mainly here to serve as a starting point. Do you either @seanpreston or @ThomasLaPiana any specific wording you'd prefer over what is currently there?

Reply:
maybe also link to the docs site? we don't have the official migration guide up yet to link to unfortunately, but I think having links to both would be helpful

Reply:
This wording works for me. Adding something to the README and docs is necessary too. @conceptualshark do you have any opinions here?